docs/source/ultimate_gdpr.rst

```diff
@@ -1,32 +1,32 @@
 
 Access to your data for verification
 ____________________________________
-Some people can access all your data at the research location. Including the data without a code. This is necessary to check whether the study is being conducted in a good and reliable manner and to be able to notify you and your physician in the case of any incidental findings resulting from your brain scans. Persons who have access to your data for review are: the local committee that monitors the safety of the study, the data controller <the University of XXX> working for the primary investigator of the study, and local, national and international supervisory authorities. They will keep your data confidential.
+Some people can access all your data at the research location, including the data without a code. This is necessary to check whether the study is being conducted in a good and reliable manner and to be able to notify you and your physician in case of any incidental findings resulting from your brain scans. Persons who have access to your data for review are: the local committee that monitors the safety of the study, the data controller <at the University of XXX> working for the primary investigator of the study, and local, national and international supervisory authorities. They will keep your data confidential.
 
 Retention period of your data
 _____________________________
 There is no plan to delete your data as they can be re-used for legitimate research interest. We will however re-evaluate every [10 years] if it is worthwhile keeping them.
 
 Withdrawing consent
 ___________________
-You can withdraw your consent to the use of your personal data. This applies to this study and also to the sharing for future research. You have however to understand that once shared with other institutions, it is impossible to remove your data from such copies.
+You can withdraw your consent to the use of your personal data at any time. This applies to this study and also to the sharing for future research. You have however to understand that once shared with other institutions, it is impossible to remove your data from such copies.
```

Comment:
We specifically left out "at any time" as this was one of our group's big discussion points. Given the GDPR circumstances and based on our experience, we voted for including a specific date, e.g. in the form of  "You can withdraw your consent to the use of your personal data until <day/month/year>." so that it can be adapted by the respective researchers according to their ethics/needs/standard (for example giving participants one month to withdraw their consent).

Reply:
ok since we use [ ] for bits people should think about/modify why not:

> You can withdraw your consent to the use of your personal data [at any time,  and this applies to the study you are taking part in and also to the sharing for future research / up to -Date- from which it will deposited in a repository]. You have however to understand that once shared with other institutions, it is impossible to remove your data from such copies.

Reply:
thx @CPernet, that sounds reasonable!

Reply:
Agreed!

Reply:
Looks good. It's worth mentioning that the reason we include a "deadline" for withdrawal is because, based on discussions at the GLiMR workshop, we consider the legal basis for processing and sharing data to be "public interest/good" or "legitimate interest of the controller". If it were based on "informed consent", the subject would still have the right to withdraw their data *at any time*, which also implies the the controller has to put infrastructure in place for allowing that.